contracts/Lens/VenusLens.sol

```diff
@@ -584,14 +584,38 @@ contract VenusLens is ExponentialNoError {
     }
 
     /**
-     * @notice Calculate the total XVS tokens pending or accrued by a user account
+     * @dev Struct for Pending Rewards for per market
+     */
+    struct PendingReward {
+        address vTokenAddress;
+        uint256 amount;
+    }
+
+    /**
+     * @dev Struct for Reward of a single reward token.
+     */
+    struct RewardSummary {
+        address rewardTokenAddress;
+        uint256 totalRewards;
+        PendingReward[] pendingRewards;
+    }
+
+    /**
+     * @notice Calculate the total XVS tokens pending and accrued by a user account
      * @param holder Account to query pending XVS
      * @param comptroller Address of the comptroller
-     * @return Total number of accrued XVS that can be claimed
+     * @return Reward object contraining the totalRewards and pending rewards for each market
      */
-    function pendingVenus(address holder, ComptrollerInterface comptroller) external view returns (uint) {
+    function pendingRewards(
+        address holder,
+        ComptrollerInterface comptroller
+    ) external view returns (RewardSummary memory) {
         VToken[] memory vTokens = comptroller.getAllMarkets();
         ClaimVenusLocalVariables memory vars;
+        RewardSummary memory rewardSummary;
+        rewardSummary.rewardTokenAddress = comptroller.getXVSAddress();
+        rewardSummary.totalRewards = comptroller.venusAccrued(holder);
+        rewardSummary.pendingRewards = new PendingReward[](vTokens.length);
         for (uint i = 0; i < vTokens.length; i++) {
```

Comment:
Gas optimisation and follow everywhere we are using for loop.
* remove re-decleration of uint i to 0
* use ++i instead of i++

Reply:
Done.